plugins/README.md

```diff
@@ -0,0 +1,31 @@
+# Collections Plugins Directory
```

Comment:
Do we need this file?

Reply:
https://github.com/ansible-collections/community.mysql/tree/815ad0de994a67f1ea1d0e24a8eac47e474b9885/plugins I don't dislike the context it adds if someone is browsing through the repo.. ..happy to remove it if you think it makes sense though 👍

Reply:
If it helps you, it will help others.